contrib/gin-gonic/gin/gintrace_test.go

```diff
@@ -554,6 +555,10 @@ func TestAppSec(t *testing.T) {
 	r.Any("/path0.0/:myPathParam0/path0.1/:myPathParam1/path0.2/:myPathParam2/path0.3/*param3", func(c *gin.Context) {
 		c.String(200, "Hello Params!\n")
 	})
+	r.Any("/body", func(c *gin.Context) {
+		pAppsec.MonitorParsedHTTPBody(c.Request.Context(), "$globals")
```

Comment:
Question: what is `"$globals"`?

Reply:
`$globals` is a pattern that will trigger an attack detection in the WAF if found in the unmarshalled body.
It's a PHP variable that references all available variables in the scope. As an example, it could be used by an attacker to dump the server PHP environment: `echo "$globals"`.

Reply:
Thanks for the context. 👍 
It may be good to document that since it's relevant to the test, but I'll leave that up to you.